drivers/modem/ublox-sara-r4.c

```diff
@@ -378,7 +378,12 @@ static ssize_t send_socket_data(void *obj,
 	}
 
 	/* Wait for prompt '@' */
-	k_sem_take(&mdata.sem_prompt, K_FOREVER);
+	ret = k_sem_take(&mdata.sem_prompt, K_SECONDS(1));
+	if (ret != 0) {
+		ret = -ETIMEDOUT;
+		LOG_ERR("No prompt sem");
```

Comment:
This error message is not clear to me. Maybe something like "No @ prompt received"?

Reply:
Done